### PR TITLE
Toast Notification Wrapper

### LINKED ISSUE #197: Toast Notification Wrapper Component
This component will wrap the `ToastNotification` component(s) such that they are aligned vertically at the top right of the browser.  We will determine in the future whether we will use a scroll bar for notifications that don't fit on the page or add a button to show additional notifications.

### PR DESCRIPTION
**Pull request by natalynyu**
The `ToastNotificationWrapper` component will be used to position the `ToastNotification` components such that they line up at the top right of the page.

The example includes a button to show the `ToastNotification` components wrapped in the `ToastNotificationWrapper` at the top right of the page. You will most likely need to scroll up to see the example, as it is purposely positioned outside of the regular example container. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality
      to change)

## Checklist:

- [x] I have run `eslint` on the code
- [x] I have added JSDoc for all of my code (where applicable)
- [ ] I have added tests to cover my changes.

## Priority:
- [x] Normal 
- [ ] High 

## Related Issues:

Fixes #197

### CODE REVIEW

File: src/Alerts/ToastNotificationWrapper.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+const ToastNotificationWrapper = styled.div`
+  display: flex;
+  align-self: flex-end;
+  flex-direction: column;
+  height: 100%;
+  overflow-y: auto;
+  position: absolute;
+  top: 5px;
+  right: 5px;
+`;
```
Comment (amalyam) [inline src/Alerts/ToastNotificationWrapper.tsx:16]:
```suggestion
z-index: 1000;
`;
```
I think we also need to set a high z-index for accessibility and usability (to ensure it remains on top of other content)

**Review by amalyam — CHANGES_REQUESTED** [review]
This looks good! In addition two the two inline comments I left, I had two other thoughts: 
 - should we have a prop like "autoHideDuration" so we can set an interval after which these will automatically close?
 - I was looking at MUI's [toast component](https://mui.com/material-ui/react-snackbar/) - they provide an "anchorOrigin prop to control the Snackbar's position on the screen". May not be necessary for now, but might be nice to include more than one option in case the default covers important content?

File: src/Alerts/ToastNotificationWrapper.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+ * A wrapper for the ToastNotification component to position them at the top
+ * right of the page.
+ */
+const ToastNotificationWrapper = styled.div`
+  display: flex;
+  align-self: flex-end;
+  flex-direction: column;
+  height: 100%;
+  overflow-y: auto;
+  position: absolute;
```
[the remark this answers is not in the document]
Comment (natalynyu) [reply src/Alerts/ToastNotificationWrapper.tsx]: Yes, that's an excellent idea! I updated the component to use `position: fixed` instead in commit [f4cb945](https://github.com/seas-computing/mark-one/pull/198/commits/f4cb945a5604d9658993c8f9b36ec34a6e46013a).

File: src/Alerts/ToastNotificationWrapper.tsx (hunk 1 above)
Comment (natalynyu) [reply src/Alerts/ToastNotificationWrapper.tsx:17]: Good point. This was updated in commit [99f6377](https://github.com/seas-computing/mark-one/pull/198/commits/99f6377d6206e410518ae2e0f048ef8e09ea07e0).

**Comment by natalynyu** [comment]
Thank you for the suggestions, @amalyam! We've discussed in the past the possibility of having notifications disappear automatically, but we had some accessibility concerns. Vittorio had advised that having content disappear on its own could be tricky, since we'd have to make a decision on how long is long enough to display the message.

**Review by amalyam — APPROVED** [review]
Everything looks great and functions as expected! Nice work incorporating all of the different suggestions!

**Pull request merged**